packages/components/src/textarea-control/types.ts

```diff
@@ -0,0 +1,29 @@
+/**
+ * External dependencies
+ */
+import type { TextareaHTMLAttributes } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import type { BaseControlProps } from '../base-control/types';
+
+export type TextareaControlProps = Pick<
+	BaseControlProps,
+	'hideLabelFromVision' | 'help' | 'label'
+> & {
+	/**
+	 * A function that receives the value of the textarea.
```

Comment:
Nitpick: Past advice has been to make sure that these comments match the readme 1:1.